projects/ontimize-web-ngx/src/lib/components/app-header/o-app-header.component.html

```diff
@@ -1,4 +1,4 @@
-<nav fxFlex fxLayout="row" fxLayoutAlign="space-between stretch">
+<nav fxFlex fxLayout="row" fxLayoutAlign="space-between stretch" [ngClass]="getColorClass()">
```

Comment:
Maybe it is better to use something like this
`[ngClass]="color ? 'o-app-header-'+color : ''"`
So we avoid method `getColorClass` will be executed all the time.